pyproject.toml

```diff
@@ -21,12 +21,7 @@ dependencies = [
   "tabulate",
   "tqdm",
   "humanfriendly",
-  # cyvcf2 also pulls in coloredlogs and click",
```

Comment:
Let's keep the comments here with the rationale for these dependencies. The only reason we're using coloredlogs is because it comes anyway with cyvcf2